scootapi/server/serve.go

```diff
@@ -20,16 +24,45 @@ func MakeServer(handler scoot.CloudScoot,
 }
 
 type Handler struct {
-	scheduler scheduler.Scheduler
-	sagaCoord saga.SagaCoordinator
-	stat      stats.StatsReceiver
+	scheduler          scheduler.Scheduler
+	sagaCoord          saga.SagaCoordinator
+	stat               stats.StatsReceiver
+	runJobStatCount    int64
+	jobStatusStatCount int64
+}
+
+func (h *Handler) loop() {
+	tickerS := time.NewTicker(time.Second)
+	tickerM := time.NewTicker(time.Minute)
+	for {
+		// Manually latch and reset until we find a better way to do req/sec within viz's one minute window.
```

Comment:
Twitter's Observability Stack Doesn't support req/sec.  Is there a reason we want this other than nice to have? I suspect our req/sec and even req/min will fairly low initially.

If we do want req/sec I can show you the hack that Observability & Manhattan use which generates a stat for every second in the minute.  You end up with stats like
req_0
req_1
req_2
...
req_59

Then you still don't need latching the are just using counters and rates in Viz.  Implementing any kind of adhoc latching is going to make this difficult to reason about if most metrics aren't latched